pkg/registry/secret/etcd/etcd_test.go

```diff
@@ -63,7 +63,7 @@ func TestCreate(t *testing.T) {
 		},
 		&api.Secret{
 			ObjectMeta: api.ObjectMeta{Name: "name"},
-			Data:       map[string][]byte{".dotfile": []byte("")},
+			Data:       map[string][]byte{"~.dotfile": []byte("")},
```

Comment:
> Ok, I see the comment above now. Renaming the case would help. ~.invalidname or something. Esp. since .dotfile is going to be legal.

Looks the library wants to test creates by using one valid example and many invalid examples, so renaming the method isn't accurate.